.github/workflows/builder.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           python3 -m pip install packaging
           python3 -m pip install --use-deprecated=legacy-resolver .
           python3 script/version_bump.py nightly
-          version="$(python setup.py -V)"
+          version="$(python -m homeassistant --version)"
```

Comment:
maybe we can just make `version="$(python3 script/version_bump.py nightly)"`

Reply:
Oh that might even be better 👍 
Good idea, will adjust.

Reply:
![image](https://user-images.githubusercontent.com/195327/155103934-dfe82dcd-5d41-4a9b-92f9-7d49387adead.png)

Done in [3ec2369](https://github.com/home-assistant/core/pull/67022/commits/3ec2369d63521a7317161b5f70969d764d37a451)